### PR TITLE
Nand nor test see #366 (Pending)

### PR DESCRIPTION
**Pull request by maxpid**
## Content

Please describe your contribution here...

---

## Checklist

Please check that your pull request is correct:

- Each commit:
    - [ ] corresponds to a contribution that should be notified to users,
    - [ ] does not generate new errors or warnings at compile or test time,
    - [ ] must be attributed to its real authors (with correct GitHub IDs and [correct syntax for multiple authors](https://help.github.com/articles/creating-a-commit-with-multiple-authors/)).
- The title of a commit should:
    - [ ] begin with a contribution type
        - `FEATURE` for a behaviour allowing a user to do something new,
        - `FIX` for a behaviour which has been changed in order to meet user’s expectations,
        - `TEST` when it concerns an acceptance test,
        - `PROCESS` for a change in the way the software is built, tested, deployed,
        - `DOC` when it concerns only internal documentation (however it is better to combine it with the contribution that required this documentation change),
    - [ ] be followed by a colon (`:`) with one space after and no space before,
    - [ ] be followed by a title (written in English) as short, as user-centered and as explicit as possible
        - If it is a feature, the title must be the user action (beginning with a verb, and please not `manage`),
        - If it is a fix, the title must describe the intended behavior (with `should`).
    - [ ] ends with a reference to the corresponding ticket with the following syntax:
        - `(closes #xx)` if xx is a feature ticket (and the commit is a complete implementation),
        - `(fixes #xx)` if xx is a fix ticket (and the commit is a complete fix),
        - `(see #xx)` otherwise,
- Each committed line is:
    - [ ] useful (it would not work if removed)
        - if it is a comment line, its information could not be conveyed by better variables and function naming, better code structuring, or better commit message,
    - [ ] related to this very contribution (feature, fix...),
    - [ ] in English (with the exception of Gherkin scenarios in French and resulting steps),
    - [ ] without any typo in variable, class or function names,
    - [ ] correctly indented (spaces rather than tabs, same number of characters as in the rest of the file).

### CODE REVIEW

**Comment by maxpid** [comment]
test

File: features/step_definitions/event.rb [hunk 1]
```diff
@@ -57,3 +57,9 @@
   end
 end
 
+Soit("{string} les rubriques sélectionnées négativement") do |topics|
+  visit '/'
+  topics.split('|').each do |topic|
+     click_on topic
```
Comment (benel) [inline features/step_definitions/event.rb:63]: Avec un seul clic c'est une sélection positive et non négative...

File: features/portfolio_browse.feature [hunk 2]
```diff
@@ -66,3 +66,10 @@ Scénario: en remplaçant le ET par un OU
   Et l'item "SM 001 n" est caché
   Et l'item "SNZ 005" est caché
   Et l'item "SR 005" est caché
+    
+  Scénario: composition ET et OU négativement
+
+    Soit "Vincent Larcher" et "XIXe s." et "XVIe s." les rubrique sélectionnées négativement
+    et "SR 005" un des items cachés
+    et "SNZ 006" un des items cachés
+    et "AXN 009" un des items cachés
```
Comment (benel) [inline features/portfolio_browse.feature:75]: Et il n'y a ni `Quand` ni `Alors` ?

**Comment by benel** [comment]
Le test et l'historique sont un peu curieux...

Mais heureusement, depuis, dans le projet, vous avez sûrement eu l'occasion de remettre en pratique tout ça...

**Pull request closed without merging**